### ISSUE TITLE
Examples doubled

### ISSUE THREAD

**Issue by mohamed-barakat** [homalg-project/PackageJanitor#17]
The examples get doubled, e.g., here

https://homalg-project.github.io/Locales/doc/chap6.html#X7A489A5D79DA9E5C

once `makedoc*.g` includes the entry `"doc"` in the list [`scan_dirs`](https://github.com/homalg-project/Locales/blob/master/makedoc.g#L15).

**Comment by zickgraf** [comment]
TLDR: you can move the `@InsertChunk` statements in your `Doc.autodoc` file to another `.autodoc` file. If you do not like that solution, we can discuss the other solutions mentioned below.

The `Doc.autodoc` file is read twice, once via  the `files` entry and once via the `scan_dirs` entry `doc`. As long as `Doc.autodoc` does not contain/include any actual content, this is not a problem. I decided to do it that way for compatibility with `CAP`, see the old version of the `makedoc.g` of `CAP`: https://github.com/homalg-project/CAP_project/blob/148441236186e5d377b264bda8a6ec0018989fb9/CAP/makedoc.g#L11-L12
`CAP` has multiple `.autodoc` files in `doc` which should all be included.

Other solutions would require the `Doc.autodoc` file and additional `.autodoc` files to be in different directories, or the `Doc.autodoc` to not end with `autodoc`.

**Comment by mohamed-barakat** [comment]
I would prefer to make the record entry `files` explicit in the `site.yml` and if not provided the `scan_dir` in its current form should do the job. Or is it important to specify the first file to read?

**Comment by zickgraf** [comment]
> Or is it important to specify the first file to read?

Yes, it is, since chapters and sections are ordered in the order they are discovered by AutoDoc.

**Comment by mohamed-barakat** [comment]
I see. However, the implication of this design means for me to 

* keep `Doc.autodoc` free of `Insert*` which would force me to replicate part of its content in other files, or
* rename `Doc.autodoc`, or
* move it to another subfolder,

all of which are suboptimal. Maybe the best solution is to let `AutoDoc` ignore duplicates.

**Comment by zickgraf** [comment]
This was easier than expected: https://github.com/gap-packages/AutoDoc/pull/248

**Comment by mohamed-barakat** [comment]
Excellent

**Comment by zickgraf** [comment]
My AutoDoc PR got merged: https://github.com/gap-packages/AutoDoc/pull/248

@mohamed-barakat Could you test if this issue is fixed?

**Comment by mohamed-barakat** [comment]
Excellent, the issue is now resolved. Thank you very much.
